src/main/tut/usage/osm.md

```diff
@@ -65,5 +65,5 @@ osm.fromORC(path) match {
 ss.stop()
 ```
 
-This approach will be particularly efficient when ran on an EMR cluster, since
+This approach will be particularly efficient when run on an EMR cluster, since
```

Comment:
Well shucks, turns out `run` is the correct past participle! [I never knew.](https://www.usingenglish.com/reference/irregular-verbs/run.html)

> I have run many marathons.